ansible/roles/test/files/ptftests/arista.py

```diff
@@ -313,6 +331,49 @@ def parse_bgp_route(self, output, expects):
 
         return set(expects) == prefixes
 
+    def get_bgp_neighbors(self):
+        neigh_bgp = {}
+        dut_bgp = {}
+        for cmd, ver in [('show ip bgp neighbors', 'v4'), ('show ipv6 bgp neighbors', 'v6')]:
+            output = self.do_cmd(cmd)
+            if ver == 'v6':
+                neigh_bgp[ver], dut_bgp[ver], neigh_bgp['asn'] = self.parse_bgp_neighbor(output, get_neigh=True, get_asn=True)
+            else:
+                neigh_bgp[ver], dut_bgp[ver] = self.parse_bgp_neighbor(output, get_neigh=True)
+
+        return neigh_bgp, dut_bgp
+
+    def change_bgp_neigh_state(self, asn, state="no shut"):
+        self.do_cmd('configure')
+        self.do_cmd("router bgp %s" % asn)
```

Comment:
>" [](start = 20, length = 1)

Could you stick to one type of quotation if there is no special reason?

Reply:
Will take care of that